app/models/consultee.rb

```diff
@@ -61,7 +62,9 @@ def default_scope
     end
   end
 
-  scope :unassigned, -> { where.not(id: PlanningApplicationConstraint.pluck(:consultee_id)) } # rubocop:disable Rails/PluckInWhere
+  scope :unassigned, lambda {
```

Comment:
Might be preferable to use the arrow syntax consistently:

```suggestion
  scope :unassigned, -> {
```